### PR TITLE
Added WTab based equivalent for some methods in WTabSet

### PR DESCRIPTION
**Pull request by JohnMcGuinness**
Addresses #1626. If this is ok then I'll add some tests.

### CODE REVIEW

**Comment by JohnMcGuinness** [comment]
@jonathanaustin should I mark the content based methods as deprecated?

**Comment by ricksbrown** [comment]
@JohnMcGuinness for this and any other PRs it would be great if you could update the changelog.

File: wcomponents-core/pom.xml [hunk 1]
```diff
@@ -272,6 +272,24 @@
...
+	  <scope>test</scope>
+	  <version>1.5</version>
+	  <exclusions>
+	   <exclusion>
+	    <groupId>org.seleniumhq.selenium</groupId>
+	    <artifactId>selenium-remote-driver</artifactId>
+	   </exclusion>
+	  </exclusions>
+	 </dependency>
 	</dependencies>
```
Comment (jonathanaustin) [inline wcomponents-core/pom.xml]: Not sure why these selenium dependencies have been added (also the indenting is a mix of space and tab)

File: wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WTabSet.java [hunk 2]
```diff
@@ -496,6 +496,19 @@ public void setActiveTab(final WComponent content) {
...
+	 * @param tab the active tab.
+	 */
+	public void setActiveTab(final WTab tab) {
+		int index = getTabIndex(tab);
+
+		if (index != -1) {
+			setActiveIndex(index);
+		}
+	}
+
```
Comment (jonathanaustin) [inline wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WTabSet.java]: What do you think...to be consistent and avoid code duplication, all the new `WTab` methods could just call the existing `content` methods like you did with `isActive` method.
```
public void setActiveTab(final WTab tab){
    setActiveTab(tab.getContent());
}
```

File: wcomponents-core/pom.xml (hunk 1 above)
Comment (JohnMcGuinness) [reply wcomponents-core/pom.xml]: Weird. I'm not sure why this is here either. I'll remove it.

File: wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WTabSet.java (hunk 2 above)
Comment (JohnMcGuinness) [reply wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WTabSet.java]: Sounds good. I'll update to do this.

**Comment by JohnMcGuinness** [comment]
This closes #1626

**Pull request merged**